src/components/form/index.tsx

```diff
@@ -0,0 +1,148 @@
+import PopUp from '#components/pop-up'
+import styled from '@emotion/styled'
+import React from 'react'
+
+function Feedback() {
+  const [isDisplayed, setDisplay] = React.useState(false)
+
+  function handleSubmit(e: React.FormEvent<HTMLFormElement>): void {
+    e.preventDefault()
+    const {
+      name,
+      raidName,
+      rate,
+      description,
+    } = e.target as typeof e.target & {
+      name: { value: string }
+      raidName: { value: string }
+      rate: { value: string }
+      description: { value: string }
+    }
+    console.log({
+      name: name.value,
+      raidName: raidName.value,
+      rate: rate.value,
+      description: description.value,
+    })
+  }
+
+  return (
+    <>
+      <$Button type="button" onClick={() => setDisplay(!isDisplayed)}>
+        Feedback
+      </$Button>
+      <$Feedback>
+        <PopUp placeInView={[isDisplayed, setDisplay]}>
+          <span>Feedback</span>
+          <form onSubmit={(e) => handleSubmit(e)}>
```

Comment:
```suggestion
          <form onSubmit={handleSubmit}>
```
it's pretty safe to do this

Reply:
i am not experienced. may i ask why? 🤓

Reply:
first of all they are equivalent, 
creating a callback that invokes the function with received argument
```
onSubmit={e => handleSubmit(e)}
```
is the same as passing the function itself
```
onSubmit={handleSubmit}
```

they are both just passed `e` as an argument

Reply:
besides that, since we wrote `handleSubmit`, we know which arguments is needs and we wrote `handleSubmit` _as_ a callback

Reply:
> 
> 
> first of all they are equivalent,
> creating a callback that invokes the function with received argument
> 
> ```
> onSubmit={e => handleSubmit(e)}
> ```
> 
> is the same as passing the function itself
> 
> ```
> onSubmit={handleSubmit}
> ```
> 
> they are both just passed `e` as an argument

so why did you request that change? 🤔

Reply:
> 
> 
> besides that, since we wrote `handleSubmit`, we know which arguments is needs and we wrote `handleSubmit` _as_ a callback

aah so the idea is to make the code more readable but still why the need to wrap handleSubmit in a callback, the event wouldnt change. also we can disable the button after submitting the form or show a loader

Reply:
the handleSubmit _is_ a callback, every function that is given as a handler or as something to trigger when something else happens is referred to as a callback. `React.useCallback` doesn't make functions into callbacks, it just _memoizes functions_ and is mainly used _for_ callbacks, as opposed to `React.useMemo` that memoizes a _value_

The change is requested for readability "OnSubmit=>HandleSubmit", also prevents registering an unneeded function every render, just a good haibit

Reply:
Both are acceptable, I leave it up to the Author, either way

Reply:
> 
> 
> the handleSubmit _is_ a callback, every function that is given as a handler or as something to trigger when something else happens is referred to as a callback. `React.useCallback` doesn't make functions into callbacks, it just _memoizes functions_ and is mainly used _for_ callbacks, as opposed to `React.useMemo` that memoizes a _value_
> 
> The change is requested for readability "OnSubmit=>HandleSubmit", also prevents registering an unneeded function every render, just a good haibit

thanks for the explanation. i think i will adapt you habit =)